tox.ini

```diff
@@ -7,7 +7,7 @@
 skipsdist = True
 usedevelop = True
 envlist =
-    py{37,38,39,310,311}-dj{32,40,41}-wt{41,42}
+    py{37,38,39,310,311}-dj{32,41}-wt{41,42}
```

Comment:
This is missing wagtail 5.0.

But also dj32 isn't supported on python 3.11 means you may need to add some extra env's here to take that into account.

e.g. https://github.com/torchbox/wagtailmedia/blob/259ac0247fade78a6d447bac968b8881053c7c5c/tox.ini#L4

Reply:
Should be changed to:
```
py{38,39,310}-dj{40}-wt{41,42}
py{38,39,310}-dj{32,41}-wt{41,42,50}
py{311}-dj{41}-wt{41,42,50}
py{311}-dj{42}-wt{50}
```